Realm.Shared/Dynamic/DynamicRealmObjectHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Realms.Weaving;
+
+namespace Realms.Dynamic
+{
+    internal class DynamicRealmObjectHelper : IRealmObjectHelper
```

Comment:
Fields are faster- quite happy if we don't have any other reason to use properties